### PR TITLE
Improvements to HTMLMediaElement

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: api/HTMLMediaElement.json [hunk 1]
```diff
@@ -87,6 +160,27 @@
                   "value_to_set": "true"
                 }
               ]
+            },
+            "chrome": {
```
Comment (ddbeck) [inline api/HTMLMediaElement.json]: Would you mind adding these entries in sorted order?

File: api/HTMLMediaElement.json [hunk 2]
```diff
@@ -305,6 +411,15 @@
               "notes": [
                 "Firefox doesn't implement this yet. See <a href='https://bugzil.la/847377'>bug 847377</a>."
               ]
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
```
Comment (ddbeck) [inline api/HTMLMediaElement.json]: Like you've done above, could you add the `null` Safari (desktop and iOS) entries?

File: api/HTMLMediaElement.json (hunk 1 above)
Comment (Elchi3) [reply api/HTMLMediaElement.json]: This is now actually required. Updated this.

File: api/HTMLMediaElement.json (hunk 2 above)
Comment (Elchi3) [reply api/HTMLMediaElement.json]: Added

**Pull request merged**